app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  load_and_authorize_resource
+  before_action :authenticate_user!
+  before_action :set_user, only: %i[ update destroy ]
+
+
+  def destroy    
+    @user.destroy
+    respond_to do |format|
+      format.html { redirect_to (request.referrer||admin_path),status: 302, notice: 'User was successfully deleted.' }   
+      format.json { head :no_content } 
+    end
+  end
+
+  def update
+    @user.update_column(:email, params[:user].delete(:email))
```

Comment:
Why are we doing this?

Reply:
Because as a admin I want do have ability to edit users email, and because there is email confirmation I can do it otherwise